docs/getting-started/1242-install.mdx

```diff
@@ -57,6 +57,10 @@ curl -L https://dl.dagger.io/dagger/install.sh | DAGGER_VERSION=0.2.19 sh
 dagger 0.2.19 (GIT_SHA) darwin/arm64
 ```
 
+:::note
+you need elevated privileges to run the previous command or append `sudo` before `sh`
```

Comment:
note: user not the case on macOS (and on Linux it depends -- on Ubuntu if the user is part of the `docker` group, not needed either)

Reply:
"elevated privileges" is not very clear to me here - could we specify what specific privileges the user needs? Or, alternatively why not update the command directly instead of using a note ie. `curl -L https://dl.dagger.io/dagger/install.sh | DAGGER_VERSION=0.2.19 sudo sh`?

Reply:
☝️ @marcosnils